MaterialComponents.podspec

```diff
@@ -37,6 +37,16 @@ Pod::Spec.new do |mdc|
   #  end
   #
 
+  mdc.subspec "schemes" do |scheme_spec|
+
```

Comment:
Nit?: Remove extra newline? One more below.

Reply:
Moved down beside private. Spacing matches the existing style.